lib/Basic/Platform.cpp

```diff
@@ -848,6 +848,37 @@ llvm::VersionTuple swift::getTargetSDKVersion(clang::DarwinSDKInfo &SDKInfo,
   return SDKVersion;
 }
 
+std::optional<llvm::Triple>
+swift::getCanonicalTriple(const llvm::Triple &triple) {
```

Comment:
Is it weird to have this in Swift and not LLVM?

Reply:
I want to avoid another revlock automerger situation at the moment.